packages/core/src/components/CardsTable/Body/Row/Row.styled.tsx

```diff
@@ -0,0 +1,23 @@
+import { styled } from '@medly-components/utils';
+import Card from '../../../Card';
+import { Props } from './types';
+
+export const Row = styled(Card)<Props>`
+    cursor: ${({ disabled, onClick }) => (disabled ? 'not-allowed' : onClick ? 'grab' : 'inherit')};
+    box-sizing: border-box;
+    border-radius: 0.8rem;
+    border: ${({ withLightTheme }) => withLightTheme ? 'default' : 'none'};
+    box-shadow: ${({ withLightTheme }) => withLightTheme ? 'none' : '0 0.2rem 0.8rem rgba(176,188,200,0.2)'};
+    
+    &:hover {
+        box-shadow: ${({ withLightTheme }) => withLightTheme ? '0 0.2rem 0.4rem rgba(176,188,200,0.60)' : '0 0.4rem 0.8rem rgba(176,188,200,0.60)'};
+    };
+
+    & > * {
+        background-color: ${({ theme, disabled }) => `${disabled ? theme.table.rowDisabledColor : 'inherit'}`};
+    };
+
+    :nth-child(2n) {
+        background: ${({ theme, withLightTheme}) => withLightTheme ? `${theme.colors.grey[50]}` : theme.table.rowBackgroundColor};
```

Comment:
we should get this property from cardTable theme

Reply:
got it